docs/install_guides/installing-openvino-apt.md

```diff
@@ -1,164 +1,133 @@
 # Install Intel® Distribution of OpenVINO™ toolkit for Linux* Using APT Repository {#openvino_docs_install_guides_installing_openvino_apt}
 
-This guide provides installation steps for Intel® Distribution of OpenVINO™ toolkit for Linux* distributed through the APT repository.
+This guide provides installation steps for Intel® Distribution of OpenVINO™ toolkit Runtime for Linux* distributed through the APT repository.
```

Comment:
Per marketing/legal, these asterisks are no longer required. They are confusing and annoy some users, I recommend removing them.

Reply:
OK, then let start to remove the asterisks from now on. I will start by updating this branch first.